Makefile

```diff
@@ -18,12 +20,47 @@ $(TARGET):
 		-o $(TARGET) \
 		./cmd/cilium
 
+release:
+	docker run \
+		--env "RELEASE_UID=$(RELEASE_UID)" \
+		--env "RELEASE_GID=$(RELEASE_GID)" \
+		--rm \
+		--workdir /cilium \
+		--volume `pwd`:/cilium docker.io/library/golang:1.15.7-alpine3.13 \
+		sh -c "apk add --no-cache make && make local-release"
+
+local-release: clean
+	for OS in darwin linux; do \
+		EXT=; \
+		ARCHS=; \
+		case $$OS in \
+			darwin) \
+				ARCHS='amd64'; \
```

Comment:
Might be outside the scope of this PR, but should we start `arm64` OSX builds?

Reply:
Yes. However, as for Hubble CLI, this is pending on Go 1.16 to be released.